### PR TITLE
improve doc for PBT tuner

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: docs/en_US/Tuner/BuiltinTuner.md [hunk 1]
```diff
@@ -463,13 +463,13 @@ tuner:
...
 
 **classArgs requirements:**
 
 * **optimize_mode** (*'maximize' or 'minimize'*) - If 'maximize', the tuner will target to maximize metrics. If 'minimize', the tuner will target to minimize metrics.
 * **all_checkpoint_dir** (*str, optional, default = None*) - Directory for trials to load and save checkpoint, if not specified, the directory would be "~/nni/checkpoint/<exp-id>". Note that if the experiment is not local mode, users should provide a path in a shared storage which can be accessed by all the trials.
-* **population_size** (*int, optional, default = 10*) - Number of trials for each step. In our implementation, one step is running each trial by specific training epochs set by users.
+* **population_size** (*int, optional, default = 10*) - Number of trials in a population. Each step has this number of trails. In our implementation, one step is running each trial by specific training epochs set by users.
```
Comment (squirrelsc) [inline docs/en_US/Tuner/BuiltinTuner.md:472]: trail should be trial?

Comment (QuanluZhang) [reply docs/en_US/Tuner/BuiltinTuner.md:472]: thanks, fixed

File: README.md [hunk 2]
```diff
@@ -108,6 +108,7 @@ Within the following table, we summarized the current NNI capabilities, we are g
             <li><a href="docs/en_US/Tuner/BuiltinTuner.md#Evolution">Naïve Evolution</a></li>
             <li><a href="docs/en_US/Tuner/BuiltinTuner.md#Anneal">Anneal</a></li>
             <li><a href="docs/en_US/Tuner/BuiltinTuner.md#Hyperband">Hyperband</a></li>
+            <li><a href="docs/en_US/Tuner/BuiltinTuner.md##PBTTuner">PBT</a></li>
```
Comment (leckie-chn) [inline README.md:111]: `##` => `#`

**Comment by leckie-chn** [comment]
also need to add entry in `.rst` file.

File: README.md (hunk 2 above)
Comment (QuanluZhang) [reply README.md:111]: good catch, thanks

**Comment by QuanluZhang** [comment]
> also need to add entry in `.rst` file.

has already added.

**Pull request merged**